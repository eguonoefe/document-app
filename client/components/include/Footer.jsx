import React from 'react';

/**
 * Footer component that displays a footer.
 * @returns {String} The HTML markup for the Footer component
 */
const Footer = () => (
  <div className="footer">
    <p className="center-align">&copy; 2017 | DMS</p>
    <p className="center-align">Developed by: Efekemo Eguonoghene</p>
  </div>
);

export default Footer;
