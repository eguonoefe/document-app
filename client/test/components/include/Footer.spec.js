/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../../../components/include/Footer';

configure({ adapter: new Adapter() });

jest.dontMock('../../../components/include/Footer');

describe('Footer Component', () => {
  test('should match the footer snapshot', () => {
    const component = shallow(<Footer />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
