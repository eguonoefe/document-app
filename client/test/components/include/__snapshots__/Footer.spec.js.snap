// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match the footer snapshot 1`] = `
<div
  className="footer"
>
  <p
    className="center-align"
  >
    Â© 2017 | DMS
  </p>
  <p
    className="center-align"
  >
    Developed by: Efekemo Eguonoghene
  </p>
</div>
`;
